conf/st2.conf.sample

```diff
@@ -73,8 +73,12 @@ backend = flat_file
 home_dir = /opt/cslang
 
 [content]
+# A URL pointing to the pack index. StackStorm Exchange is used by default. Use a comma-separated list for multiple indexes if you want to get other packs discovered with "st2 pack search".
+index_url = https://index.stackstorm.org/v1/index.json # comma separated list allowed here.
```

Comment:
See https://github.com/StackStorm/st2/pull/2842/files#diff-3a3bf900cae2541d75a791bc40814158R66

It works from right to left (the last one has priority), but could be changed to left-to-right. Pack names are assumed to be unique, and if you have overlapping pack names in different indexes, they'll be overwritten. I.e. if I maintain my own fork of the `sensu` pack and keep the name, my fork will be pulled when I do `st2 install sensu` if my index comes last in the list.

We should definitely document this, because it's also useful when maintaining local mirrors, etc.